hs_core/authentication.py

```diff
@@ -29,6 +29,17 @@ def create_user(self, claims):
             state=claims.get('state', ''),
             subject_areas=subject_areas)
 
+    def describe_user_by_claims(self, claims):
```

Comment:
@sblack-usu can you explain why we have to override these?
https://github.com/mozilla/mozilla-django-oidc/blob/main/mozilla_django_oidc/auth.py#L73-L82

Reply:
This just filters the user by username instead of email. This is needed because email may change and username can not.

Reply:
nice, thanks